domain-management/src/main/java/org/jboss/as/domain/management/parsing/AuditLogXml_4.java

```diff
@@ -639,175 +633,4 @@ private void parseAuditLogHandlersReference(final XMLExtendedStreamReader reader
             }
         }
     }
-
-    @Override
-    public void writeAuditLog(XMLExtendedStreamWriter writer, ModelNode auditLog) throws XMLStreamException {
```

Comment:
Should this be removed? It looks like the `StandaloneXml_5` and `HostXml_5` may invoke this and get a `UnsupportedOperationException`.